blockchain/blockchain.go

```diff
@@ -1397,6 +1415,41 @@ func (bc *BlockChain) gcCachedNodeLoop() {
 	}()
 }
 
+func (bc *BlockChain) pruneTrieNodeLoop() {
+	// ReadPruningMarks(1, limit) is very slow because it iterates over the most of MiscDB.
+	// ReadPruningMarks(start, limit) is much faster because it only iterates a small range.
+	startNum := uint64(1)
```

Comment:
This is a minor comment.

By replacing the hardcoded value with a command line flag, we can preserve certain ranges. This straightforward modification allows for the retention of specific old states that are required. Implementing pruning in conjunction with archive mode would further enhance the effectiveness of the process. Compared to the standard practice of committing every 128 blocks (default interval)

Reply:
That could be a useful feature - to preserve only a certain range of blocks - but we need to consider how this feature can be used before implementing it. Thanks.